soh/soh/OTRGlobals.cpp

```diff
@@ -323,6 +332,32 @@ OTRGlobals::OTRGlobals() {
 OTRGlobals::~OTRGlobals() {
 }
 
+void OTRGlobals::ScaleImGui() {
+    float scale = imguiScaleOptionToValue[CVarGetInteger("gImGuiScale", defaultImGuiScale)];
+    float newScale = scale / previousImGuiScale;
```

Comment:
I'm not sure I understand the point of keeping track of the previous scale value. Shouldn't this scaling only ever be applied to the defaultImGuiScale?

As it is, it seems like if you switched from default to large, then from large to extra large, you'd end up with a different value than if you just went from default to extra large. Or am I misunderstanding?

Reply:
No, ImGui itself applies the scaling to the current scale. If you tried to just apply the `scale` value as is, if you selected the one that did 2 from the .75, ImGui would end up with 1.5, or if you had 1.5 and selected the 2, ImGui would make it the equivalent of a 3.

Reply:
As far as I know, there's no "ResetScale" function that would make direct application of the scale value viable.

Reply:
Aha I see, so this is your way of working around ImGui doing relative scaling.

Any other plans/thoughts here? Otherwise I'm going to try it out and drop an approval

Reply:
It does seem like there is some issues around this. Maybe not at the fault of your code but potentially rounding? If you swap between "small" and "extra large" a few times, then back to normal, it's definitely not getting back to what normal should be

Reply:
Yeah, that's the unfortunate nature of relative scaling here. I don't know of any way of prevent that, you just have to restart SoH to normalize it. I did it on the assumption that 99% of people wouldn't be doing that repeated changing in the same session, especially between the smallest and largest options.

As for other thoughts, nothing pertaining to this PR. I want to get dynamic font size based on UI scale in at some point, but that's going to require a major refactor of LUS, if it's even possible the way I want to do it.

Reply:
Also, the reason I don't just do 2.25 instead of 2 at the top is because of the current bug surrounding the padded enhancement sliders not properly resizing with certain values and making that expanding slider visual when opening menus.